libwallet/assets/dcr/dex_wallet.go

```diff
@@ -33,17 +33,29 @@ import (
 
 // DEXWallet wraps *Asset and implements dexdcr.Wallet.
 type DEXWallet struct {
-	*Asset
+	w                    *dcrwallet.Wallet
+	helper               WalletHelper
 	tradingAccountNumber int32
 	syncData             *SyncData
 }
 
+type WalletHelper interface {
+	AccountName(accountNumber int32) (string, error)
+	IsAccountMixerActive() bool
+	UnmixedAccountNumber() int32
+	MixedAccountNumber() int32
+	LockWallet()
+	UnlockWallet(pass string) error
+}
```

Comment:
*dcrwallet.Wallet has methods for `AccountName`, `LockWallet` and `UnlockWallet`. Why are we using a different implementation's?

Reply:
https://github.com/crypto-power/cryptopower/pull/438#discussion_r1474489703